frozenlist/__init__.py

```diff
@@ -18,6 +19,12 @@ class FrozenList(MutableSequence):
 
     __slots__ = ('_frozen', '_items')
 
+    try:
+        # Python 3.9 PEP 585
+        __class_getitem__ = classmethod(types.GenericAlias)
+    except AttributeError:
+        pass
```

Comment:
Could you please just return `self` if GenericAlias doesn't exist?

Another thing is the exception usage here.
The code is 100% correct but `if sys.version_info >= (3, 9):` provides more information to reader what is the expected version with desired behavior and works much better with `mypy` and other type checkers.